stdlib/3.4/enum.pyi

```diff
@@ -7,11 +7,14 @@ class Enum:
     def __dir__(self) -> List[str]: ...
     def __format__(self, format_spec: str) -> str: ...
     def __hash__(self) -> Any: ...
+    def __iter__(self) -> Iterator['Enum']: ...
```

Comment:
I think that the type object is iterable, not the instance. I wonder if this should be a static or class method? (I'm not sure how mypy would deal with this, though.) Also, the iterator should iterate over the type of the `Enum` subclass instead of the base class, but we can't represent that right now. I'd suggest using `Iterator[Any]` as the return type and adding a comment about how it should work.